tests/BasicSearchTests.FunctionalTests.Core/Constants.cs

```diff
@@ -6,7 +6,8 @@ namespace BasicSearchTests.FunctionalTests.Core
     public static class Constants
     {
         // Predefined Texts
-        public const string ExistentSearchString = "Newtonsoft.Json";
+        public const string TestPackageId = "BaseTestPackage";
```

Comment:
Will do! We decided to not test Newtonsoft.Json as the author could break our tests by simply changing the package's metadata. We avoid this potential issue by using a package that we control.